helm_deploy/laa-estimate-eligibility/templates/ingress.yaml

```diff
@@ -35,3 +39,16 @@ spec:
                 port:
                   name: http
   {{- end }}
+    {{- if .Values.metabase.enabled }}
+    - host: "ccq-dashboard-staging.cloud-platform.service.justice.gov.uk"
```

Comment:
As this file gets used for UAT, staging and production environments, this will try to register the same host in every environment. Which becomes a problem when metabase is enabled in more than one. Can we interpolate e.g. .Values.cfe.environment_name in place of "staging" here to make it different for each environment? Or define the metabase host name in the values files?